### PR TITLE
[connectiocontext] Add prefered property

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/qofonoconnectioncontext.cpp [hunk 1]
```diff
@@ -149,6 +151,11 @@ bool QOfonoConnectionContext::active() const
     return getBool("Active");
 }
 
+bool QOfonoConnectionContext::preferred() const
+{
+    return getBool("Prefered");
```
Comment (dcaliste) [inline src/qofonoconnectioncontext.cpp]: There is an 'r' missing, I think.

Comment (neochapay) [reply src/qofonoconnectioncontext.cpp]: fixed

**Comment by monich** [comment]
We don't have such a property and probably never will. It doesn't make sense since we only have one context of each type.

**Comment by pvuorela** [comment]
By the discussion doesn't sound like this is moving forward.

**Pull request closed without merging**